### DISCUSSION TITLE
Would you use requests queueing?

### DISCUSSION THREAD

**Discussion by cristipufu** [cristipufu/aspnetcore-redis-rate-limiting#18] [Polls]
We have a working PoC of queueing in the concurrency rate limiter. Should we implement queueing for the other rate limiting strategies as well?

**Comment by gorillapower** [comment]
This is exactly what I need for my use case. There is queueing on the System.Threading.Ratelimiting APIs for TokenBucket, etc so I guess it would be nice to offer the same features. This would be suit my use case as I dont want to handle exceptions and prefer not to manually handle the queueing. bump!

Edit:
As a workaround, this can be solved quite nicely with Polly WaitAndRetry Policy. https://github.com/Polly-Contrib/Polly.Contrib.WaitAndRetry/blob/master/README.md
